config/environments/production.rb

```diff
@@ -130,12 +130,12 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: protocol + ENV["APP_DOMAIN"].to_s }
   ActionMailer::Base.smtp_settings = {
-    address: "smtp.sendgrid.net",
-    port: "587",
-    authentication: :plain,
-    user_name: "apikey",
-    password: ENV["SENDGRID_API_KEY"],
-    domain: ENV["APP_DOMAIN"],
+    address: ENV["SMTP_ADDRESS"],
+    port: ENV["SMTP_ADDRESS"],
+    authentication: ENV["SMTP_AUTHENTICATION"],
```

Comment:
same thing about falling back to `:plain`

```suggestion
    authentication: ENV["SMTP_AUTHENTICATION"].presence || :plain,
```